.rubocop_todo.yml

```diff
@@ -17,7 +17,7 @@ Metrics/AbcSize:
 # Offense count: 220
 # Configuration parameters: AllowURI, URISchemes.
 Metrics/LineLength:
-  Max: 135
+  Max: 138
```

Comment:
Should we maybe disable this cop, rather than having it very long for backwards compatibility?